internal/chain/beacon/sync_manager.go

```diff
@@ -146,7 +146,7 @@ func (s *SyncManager) Run() {
 			cancel()
 			return
 		case request := <-s.newReq:
-			_, span := metrics.NewSpanFromSpanContext(ctx, request.spanContext, "syncManager.AddCallback")
+			_, span := tracer.NewSpanFromSpanContext(ctx, request.spanContext, "syncManager.AddCallback")
```

Comment:
is there a reason this isn't `ctx, span :=` and the `ctx` passed to `s.Store.Last(ctx)` below?